docs/snippets/vue/page-story-with-args-composition.vue3.js.mdx

```diff
@@ -0,0 +1,30 @@
+```js
+// YourPage.stories.js
+
+import DocumentScreen from './YourPage.vue';
+
+//👇 Imports the required stories
+import * as PageLayoutStories from './PageLayout.stories';
+import * as DocumentHeaderStories from './DocumentHeader.stories';
+import * as DocumentListStories from './DocumentList.stories';
+
+export default {
+  component: DocumentScreen,
+  title: 'DocumentScreen',
+};
+
+const Template = (args, { argTypes }) => ({
```

Comment:
Should we remove the `{ argTypes }` from these Vue 3 signatures since they aren't used?
```suggestion
const Template = (args) => ({
```

Reply:
@phated fixed with the latest commit. Thanks for the catching that.